### ISSUE TITLE
Option to use a non default map layer

### ISSUE THREAD

**Issue by Romeo-Golf** [kevinabrandon/AboveTustin#9]
So while my use case is to use a map layer with as little geographical information as present for testing, debugging and sharing of screenshots, I figured this may be useful to other people who may wish to use a non default map layer such the Sectional and IFR charts..  would this be possible?  ;)

**Comment by kevinabrandon** [comment]
Yeah, I think at the beginning when the "browser" is set up, it clicks the "reset map" button followed by the "zoom" button.  I bet it would be pretty easy to add an option for it to click the map layer button as well...
